presto-tests/src/test/java/com/facebook/presto/tests/TestLocalQueries.java

```diff
@@ -39,6 +40,7 @@ public static LocalQueryRunner createLocalQueryRunner()
                 .setCatalog("local")
                 .setSchema(TINY_SCHEMA_NAME)
                 .setSystemProperty(REORDER_JOINS, "true")
+                .setSystemProperty(PUSH_AGGREGATION_THROUGH_JOIN, "true")
```

Comment:
I don't know. IMO in this case, enabling here has more advantages then leaving it disabled. Turning this on will likely to test all test cases, while disabling it will never test partial aggregations under joins.

Ideally, we should test both versions, but this is a topic for another story: how do we want to test features? Testing all combination is impossible, but maybe we want `TestLocalQueries` with all optional features disabled and another one "feature reach" `TestLocalQueries` with all optional features enabled?

cc @martint